src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--tue calendar--month-length--31">
+      <div class="day"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
```
day should be an element of calendar